### PR TITLE
skip id3 tag

### PR DESCRIPTION
**Pull request by mat-hek**

### CODE REVIEW

File: lib/membrane_mp3_mad_plugin/decoder.ex [hunk 1]
```diff
@@ -45,6 +62,48 @@ defmodule Membrane.MP3.MAD.Decoder do
...
+          {:skipped, payload}
+      end
+
+    with {:tag, tag_size, content} <- result do
+      tag_size =
+        tag_size
+        |> :binary.bin_to_list()
+        |> Enum.reverse()
+        |> Enum.with_index()
+        |> Enum.reduce(0, fn {el, index}, acc -> acc ||| el <<< (index * 7) end)
```
Comment (varsill) [inline lib/membrane_mp3_mad_plugin/decoder.ex:98]: We can move it to some external function that does that, as it is slightly enigmatic - we can call that function `skip_most_significant_bits` or something like that (as I believe that is what it does :D )

**Review by varsill — CHANGES_REQUESTED** [review]
One general thought - don't we want to support also ID3v1 tags (the ones put at the end of .mp3) as we are already messing around with that stuff? :D

File: lib/membrane_mp3_mad_plugin/decoder.ex [hunk 2]
```diff
@@ -45,6 +62,48 @@ defmodule Membrane.MP3.MAD.Decoder do
...
+        <<"ID3", 3::integer, _minor::integer, _flag_bytes::size(8), tag_size::binary-size(4),
+          content::binary>> ->
+          {:tag, tag_size, content}
+
+        <<"ID3", 4::integer, _minor::integer, _unsynchronisation::size(1),
+          _extended_header::size(1), _experimental::size(1), _footer::size(1), _unused::size(4),
+          tag_size::binary-size(4), content::binary>> ->
+          {:tag, tag_size, content}
+
+        payload when byte_size(payload) < 10 ->
```
[the remark this answers is not in the document]
Comment (mat-hek) [reply lib/membrane_mp3_mad_plugin/decoder.ex:84]: You mean we could have an mp3 that is less than 10 bytes long? Not sure, though I doubt it's a real case :P

**Comment by mat-hek** [comment]
@varsill it seems you know way more about ID3 than me :D I just copied the pattern matches from https://hexdocs.pm/id3vx/Id3vx.html, and it seems it doesn't handle ID3v1, so I think we can wait until we encounter that ;)

**Review by varsill — APPROVED** [review]
LGTM 🥇

**Pull request merged**